x-pack/plugins/ml/public/shared_components/single_metric_viewer/index.tsx

```diff
@@ -17,14 +20,16 @@ const SingleMetricViewerLazy = dynamic(async () => import('./single_metric_viewe
 export const getSingleMetricViewerComponent = (
   coreStart: CoreStart,
   pluginStart: MlDependencies,
-  mlServices: SingleMetricViewerServices
+  mlServices: SingleMetricViewerServices,
+  api?: SingleMetricViewerEmbeddableApi
```

Comment:
api is optional here, but in `TimeSeriesExplorerEmbeddableChart` it is non-optional

Reply:
Updated in 41c37ae127f9a44b4f83250e2b3fe1def6953a11